apple/Omnivore.xcodeproj/project.pbxproj

```diff
@@ -959,15 +959,17 @@
 /* End PBXResourcesBuildPhase section */
 
 /* Begin PBXShellScriptBuildPhase section */
-		04B8B00426A0C19900D3ED0C /* ShellScript */ = {
+		04B8B00426A0C19900D3ED0C /* Format and Lint */ = {
 			isa = PBXShellScriptBuildPhase;
+			alwaysOutOfDate = 1;
```

Comment:
Neat. Didn't know this was an option here.

Reply:
It was added in Xcode 14.2 (IIRC), along with a related warning for any build phases that didn't have input files that could be used for dependency analysis.